model/generators/markets.py

```diff
@@ -250,10 +203,122 @@ def historical_returns(self, sample_size):
         historical log-returns"""
         # TODO Consider different sampling options
         # TODO Random Seed
-        data, length = (data_feed.data, data_feed.length)
+        data, assets, length = (data_feed.data, data_feed.assets, data_feed.length)
+
         if self.model == MarketPriceModel.HIST_SIM:
             data = data[np.random.randint(low=0, high=length - 1, size=sample_size), :]
-
-        self.increments = {"cusd_usd": data[:, 0], "celo_usd": data[:, 1]}
+        increments = {}
+        for index, asset in enumerate(assets):
+            increments[asset] = data[:, index]
+        self.increments = increments
 
         logging.info("Historic increments created")
+
+
+class QuantLibWrapper():
```

Comment:
I would stick with 1 class per file. Let's move this to `model/utils/quantlib_wrapper.py`

Reply:
Sounds good